app/views/ajax/newsdata.html.erb

```diff
@@ -6,10 +6,17 @@
 <br>
 
 <div id='crud_operations_div'>
-  <b><label>Valid options:  all, title, link, description</label></b>
-  <input id="newsdata_api_word">
-
-  <br>
+  <div class="field">
+  <b><label for="newsdata_api_word">Choose a Search Option :</label></b>
+
+    <select name="newsdata_api_word" id="newsdata_api_word">
```

Comment:
Check out the rails helper for creating select fields, I think it makes sense to use rails library methods where possible.

https://guides.rubyonrails.org/v2.3/form_helpers.html#making-select-boxes-with-ease